checks/contributors.go

```diff
@@ -38,31 +36,25 @@ func init() {
 
 // Contributors run Contributors check.
 func Contributors(c *checker.CheckRequest) checker.CheckResult {
-	contribs, _, err := c.Client.Repositories.ListContributors(c.Ctx, c.Owner, c.Repo, &github.ListContributorsOptions{})
+	contribs, err := c.RepoClient.ListContributors()
 	if err != nil {
 		e := sce.Create(sce.ErrScorecardInternal, fmt.Sprintf("Client.Repositories.ListContributors: %v", err))
 		return checker.CreateRuntimeErrorResult(CheckContributors, e)
 	}
 
 	companies := map[string]struct{}{}
 	for _, contrib := range contribs {
-		if contrib.GetContributions() < minContributionsPerUser {
+		if contrib.NumContributions < minContributionsPerUser {
 			continue
 		}
-		u, _, err := c.Client.Users.Get(c.Ctx, contrib.GetLogin())
-		if err != nil {
-			e := sce.Create(sce.ErrScorecardInternal, fmt.Sprintf("Client.Users.Get: %v", err))
-			return checker.CreateRuntimeErrorResult(CheckContributors, e)
-		}
-		orgs, _, err := c.Client.Organizations.List(c.Ctx, contrib.GetLogin(), nil)
-		if err != nil {
-			c.Dlogger.Debug("unable to get org members for %s: %v", contrib.GetLogin(), err)
-		} else if len(orgs) > 0 {
-			companies[*orgs[0].Login] = struct{}{}
-			continue
+
+		for _, org := range contrib.Organizations {
+			if org.Login != "" {
+				companies[org.Login] = struct{}{}
+			}
 		}
 
-		company := u.GetCompany()
+		company := contrib.Company
 		if company != "" {
```

Comment:
is this null check even needed now ?

Reply:
It is. GitHub can return empty strings and not error.